pages/common/docker-swarm.md

```diff
@@ -0,0 +1,32 @@
+# docker swarm
+
+> A container orchestration tool.
+> More information: <https://docs.docker.com/engine/swarm/>.
+
+- Initialize a swarm cluster:
+
+`docker swarm init`
+
+- Get join token for master or worker:
```

Comment:
I think is the better `get join` could be read as `joins` something to the cluster:
```suggestion
- Display the token to join a manager or a worker:
```
I didn't see the `master` word in the docs, only `manager`: https://docs.docker.com/engine/reference/commandline/swarm_join-token/